compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc

```diff
@@ -0,0 +1,64 @@
+== Scan Images in GitLab Container Registry
+
+Use GitLab Personal access token method to authenticate Prisma to access the GitLab Container Registry.
+Added support for scanning GitLab Container Registry without having to use admin credentials to manage and get a full list of all container registries/images.
+
+[.task]
+=== Create a New Registry Scan
+
+*Prerequisties*
+
+* xref:../../install/deploy-defender/defender_types.adoc[Install a Defender] somewhere in your environment.
+* Create a https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#personal-access-token-scopes[GitLab Personal access token] with at least "read_api" scope permission.
+* Create xref:../../authentication/credentials-store/gitlab-credentials.adoc[GitLab credentials using API token].
+
+[.procedure]
+. Log in to Console, and select *Defend > Vulnerabilities > Images > Registry settings*.
+
+. Select *Add registry*.
++
+image::gitlab-container-registry.png[scale=60]
+
+. In *Add New Registry*, enter the following values:
+
+.. In *Version*, select *Gitlab Container Registry*.
+
+.. In *Registry*, specify the address to scan.
+
+.. In *Repository*, enter the name of the repository to scan, or leave this blank to scan all repositories.
+
+.. Optionally enter the *Repositories to exclude* them from being scanned.
+
+.. In *Repository types*, select the repository types that Prisma Cloud should scan.
+
+.. Enter *Tag* numbers to scan, leave blank, or enter a wildcard (*) to scan all the tags.
+
+.. Optionally, enter *Tags to exclude*, to avoid scanning images with specified tags.
+
+.. Provide at least one of the fields:
+... *User ID*: GitLab user account
+... *Group ID*: Enter a single group ID, or a list of group IDs
+... *Project ID*: Enter a GitLab Project ID, or a list of project IDs
++
+NOTE: To scan all the registries, enter only the *User ID*. When Project ID or Group ID are provided, the registry scan will be triggered accordingly. 
```

Comment:
Also mention - when all the 3 are given, the User ID takes precedence.

Reply:
Discard the above. Corrected sentence: "When we have all together- we will query the project and the group ids (will not use the user id)"